Bluepill-cli/Bluepill-cli/Simulator/SimulatorRunner.m

```diff
@@ -230,6 +301,20 @@ - (BOOL)installApplicationAndReturnError:(NSError *__autoreleasing *)error {
     return YES;
 }
 
+- (BOOL)uninstallApplicationAndReturnError:(NSError *__autoreleasing *)error {
```

Comment:
So rather than merging these duplicate methods as we previously discussed, it seems we should just locate them in different classes. The mixing of the static methods that can act on anything and the config-specific methods that know the current context is what's confusing. Confining them to separate files would be the way forward, I believe. @oliverhu? @ob?